src/ContributionsRadarChart.tsx

```diff
@@ -36,6 +37,29 @@ function ContributionsRadarChart({ author, startDate, endDate }: Props) {
 
   if (loading) return <div>Loading</div>;
 
+  /* If data does not return due to asking for over a year, show a fail state
+   * Future optimization should be to page through year at a time and
+   * combine, but this prevents a page crash
+   */
+  if (!data.user && differenceInDays(endDate, startDate) > 365) {
```

Comment:
I didn't do this for now because it's a quick api call and it isn't a hard 365 day limit. It seems like the behavior is from the first of the month thats 365 days ago - but thats a reverse engineered guess, the error message says a year is limit.